src/pages/FarmV4/FarmRow.tsx

```diff
@@ -84,7 +85,11 @@ const FarmRow: FC<{ pool: GAMMAPoolWithUserLiquidity }> = ({ pool, ...props }):
           font-poppins text-tiny font-semibold dark:text-grey-8 text-black-4
           border-grey-1 bg-grey-5 dark:bg-black-2 rounded-[2.5px] h-[25px] px-1 ml-2"
         >
-          {numberFormatter(0.2)}%
+          {(
+            new BigNumber(pool?.['latestDynamicFeeRate'] || 0.0).div(10 ** 4).toNumber() ||
+            new BigNumber(pool?.config.tradeFeeRate || 0.0).div(10 ** 4).toNumber()
```

Comment:
Why are we doing this: .div(10 ** 4) for all pools?

Reply:
In the smart contract we store these values upto 6 decimal accuracy. So to convert this in percentage we divide by 10^4 instead of 10^6